src/__tests__/generateConfluenceMarkup.test.ts

```diff
@@ -27,7 +27,7 @@ const fakeIssues: Issue[] = [
 
 const singleIssue: Issue[] = [
     {
-        title: 'ToC: links to markdown headings',
+        title: 'ToC: links to markup headings',
```

Comment:
In this case, the title was whatever the actual sample issue had, so it doesn't matter to us whether it was markup, markdown, or something else. To avoid confusion, let's delete the word "markup".